src/Frontend/Modules/Quotes/Layout/Widgets/AllQuotes.html.twig

```diff
@@ -1,4 +1,7 @@
-{% for quote in quotes %}
-  <blockquote>{{ quote.quote|raw }}</blockquote>
-  <p>{{ quote.name }}</p>
-{% endfor %}
+{% import 'Modules/Quotes/Layout/Templates/Quote.html.twig' as macro %}
```

Comment:
`macro` is too general as a name for your macro. What if you also need to import a social media buttons macro?

Reply:
fixed in b0c933b